stable/localvolumeprovisioner/templates/storageclass.yaml

```diff
@@ -4,8 +4,8 @@ kind: StorageClass
 metadata:
   name: localvolumeprovisioner
   annotations:
-    storageclass.kubernetes.io/is-default-class: {{ .Values.storageclassdefault | default "true" | quote }}
+    storageclass.kubernetes.io/is-default-class: {{ .Values.storageclass.isDefault | default "true" | quote }}
     kubernetes.io/description: Local volume provisioner StorageClass
 provisioner: kubernetes.io/no-provisioner
-volumeBindingMode: WaitForFirstConsumer
-reclaimPolicy: Delete
+volumeBindingMode: {{ .Values.storageclass.volumeBindingMode | default "Delete" | quote }}
```

Comment:
you don't need to add the defaults here, that is where values.yaml should be used for.